### PR TITLE
feat(playground): plumb through and apply template variables

### LINKED ISSUE #5034: [playground] apply template variables when invoking the template
pass down template format in the graphql layer as well as the JSON field for the template variable values and apply them before invocation

### PR DESCRIPTION
**Pull request by axiomofjoy**
Applies template variables.

https://github.com/user-attachments/assets/b523b5de-d2f8-4efb-9047-27369e8f9da1

resolves #5034

### CODE REVIEW

File: app/src/components/templateEditor/constants.ts [hunk 1]
```diff
@@ -10,6 +10,6 @@
  * ```
  */
 export const TemplateLanguages = {
-  FString: "f-string", // {variable}
-  Mustache: "mustache", // {{variable}}
+  FString: "F_STRING", // {variable}
+  Mustache: "MUSTACHE", // {{variable}}
```
Comment (axiomofjoy) [inline app/src/components/templateEditor/constants.ts:14]: Changed these to be the same as the enums in the GraphQL layer.

File: tests/utilities/test_template_formatters.py [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+        pytest.param(
+            MustacheTemplateFormatter,
+            "{{ hello }} + {{hello}} = {{ world }}",
+            {"hello": "1", "world": "2"},
+            "1 + 1 = 2",
+            id="mustache-duplicate-variables",
+        ),
+        pytest.param(
+            MustacheTemplateFormatter,
+            "{{ hello }}, {{ world }}",
```
Comment (cephalization) [inline tests/utilities/test_template_formatters.py:59]: Does the mustache formatter support escaping?

I have it setup in the frontend to escape `\{{variable}}`, removing the slash during format

**Review by cephalization — APPROVED** [review]
Looks good but I'm a little worried about formatter/syntax highlighting drift

File: tests/utilities/test_template_formatters.py (hunk 2 above)
Comment (axiomofjoy) [reply tests/utilities/test_template_formatters.py:59]: Good call, will add.

**Comment by axiomofjoy** [comment]
> Looks good but I'm a little worried about formatter/syntax highlighting drift

If it becomes a problem, can definitely move the formatting to the client

**Pull request merged**